### PR TITLE
Fix HTML display of fact tables.

### PR DESCRIPTION
**Pull request by gislenius**

### CODE REVIEW

**Comment by gislenius** [comment]
http://edinet-dev.28.io/v1/_queries/public/api/facttable-for-component.jq?token=foobar&aid=S1003U6E&format=html&role=http%3A%2F%2Fdisclosure.edinet-fsa.go.jp%2Frole%2Fjplvh%2Frol_InformationAboutFilers&profile-name=japan

File: queries/modules/io/28/apps/api.jq [hunk 1]
```diff
@@ -206,17 +206,39 @@ declare function api:csv-to-html(
     $csv as string) as item*
 {
     let $csv as string* := $csv
-    let $rows := tokenize($csv, "\n")
+    let $rows :=
+      let $newline-tokens :=
+        for tumbling window $token as string* in tokenize($csv, "\"")
+        start when true
+        end $u at $j next $v when ($j mod 2 eq 1 and $u ne "")
+                             or ($j mod 2 eq 0 and $v ne "")
```
Comment (dknochen) [inline queries/modules/io/28/apps/api.jq]: crazy fix :) Can you add a comment to explain what it does?

File: queries/modules/io/28/apps/api.jq [hunk 2]
```diff
@@ -206,17 +206,39 @@ declare function api:csv-to-html(
...
+          end $u at $j next $v when ($j mod 2 eq 1 and $u ne "")
+                               or ($j mod 2 eq 0 and $v ne "")
+          count $k
+          return if($k mod 2 eq 1)
+                 then $token ! replace($$, ",", "______28comma______")
+                 else string-join($token, "\"")
+        return tokenize(string-join($comma-tokens), "______28comma______")
+      return switch(true)
+             case contains($cell, "http://") and
+                  (contains($cell, "28.io") or
```
Comment (dknochen) [inline queries/modules/io/28/apps/api.jq]: should we replace this with request:server-name()? "28.io" shouldn't be hardcoded

**Comment by gislenius** [comment]
@dknochen makes sense, back to you!

**Pull request merged**